### PR TITLE
[Unity][CUTLASS]  Fix circular import bug in relax cutlass backend

### PR DESCRIPTION
**Pull request by digital-nomad-cheng**
There exists circular import bug in `tvm/python/tvm/relax/backend/contrib/cutlass.py`

Code to reproduce:
```
from tvm.contrib.cutlass import (
    has_cutlass,
    num_cutlass_partitions,
    finalize_modules
)
```

Error:      
```
File "/home/work/tmp/test.py", line 1, in <module>
    from tvm.contrib.cutlass import (
  File "/home/tvm/python/tvm/contrib/cutlass/__init__.py", line 18, in <module>
    from .build import has_cutlass, num_cutlass_partitions, finalize_modules, finalize_modules_vm
  File "/home/tvm/python/tvm/contrib/cutlass/build.py", line 28, in <module>
    from tvm import relax, relay, runtime
  File "/home/tvm/python/tvm/relax/__init__.py", line 90, in <module>
    from . import backend
  File "/home/tvm/python/tvm/relax/backend/__init__.py", line 19, in <module>
    from . import contrib
  File "/home/tvm/python/tvm/relax/backend/contrib/__init__.py", line 20, in <module>
    from .cutlass import partition_for_cutlass
  File "/home/tvm/python/tvm/relax/backend/contrib/cutlass.py", line 24, in <module>
    from tvm.contrib.cutlass.build import is_shape_valid_for_cutlass_matmul
ImportError: cannot import name 'is_shape_valid_for_cutlass_matmul' from partially initialized module 'tvm.contrib.cutlass.build' (most likely due to a circular import) (/home/tvm/python/tvm/contrib/cutlass/build.py)
```

### CODE REVIEW

**Comment by masahi** [comment]
I think it is better to remove `from .cutlass import partition_for_cutlass` from `backend/contrib/__init__.py` since doing `import relax` shouldn't cause `contrib/cutlass/build.py` to depend on `contrib/cutlass.py`. cc @yelite

**Comment by digital-nomad-cheng** [comment]
Import removed from `__init__`

File: python/tvm/relax/backend/contrib/__init__.py [hunk 1]
```diff
@@ -17,4 +17,3 @@
 
 """External backend codegen modules for Relax."""
```
Comment (masahi) [inline python/tvm/relax/backend/contrib/__init__.py:19]: You can remove this file now that it is not used at all.

Comment (digital-nomad-cheng) [reply python/tvm/relax/backend/contrib/__init__.py]: Thanks for the feedback, file removed.

**Pull request merged**